### PR TITLE
[PRM-943]: added SDES notification, added appealResponse to submit appeal, inclu…

### PR DESCRIPTION
**Pull request by BackRowOla**
…ded uploadFields in UploadJourney

### CODE REVIEW

**Comment by BackRowOla** [comment]
Accompanying PR's:
[Penalties-appeals](https://github.com/hmrc/penalties-appeals-frontend/pull/225)
[Penalties-stub](https://github.com/hmrc/penalties-stub/pull/23)

File: app/controllers/AppealsController.scala [hunk 1]
```diff
@@ -137,28 +140,57 @@ class AppealsController @Inject()(appConfig: AppConfig,
...
+
+  def createSDESNotifications(optUploadJourney: Option[Seq[UploadJourney]], caseID: String): Seq[SDESNotification] = {
+    optUploadJourney match {
+      case Some(uploads) => uploads.flatMap { upload =>
+        upload.uploadDetails.map { details =>
+          val uploadAlgorithm = if (upload.uploadFields.isDefined) upload.uploadFields.get("x-amz-algorithm") else ""
+          SDESNotification(
+            informationType = "S18",
+            file = SDESNotificationFile(
+              recipientOrSender = "123456789012", // may need to be changed later on
```
Comment (AndrewRButcher) [inline app/controllers/AppealsController.scala:182]: Can you make this configurable via appConfig?

File: app/controllers/AppealsController.scala [hunk 2]
```diff
@@ -137,28 +140,57 @@ class AppealsController @Inject()(appConfig: AppConfig,
...
     }
   }
+
+  def createSDESNotifications(optUploadJourney: Option[Seq[UploadJourney]], caseID: String): Seq[SDESNotification] = {
+    optUploadJourney match {
+      case Some(uploads) => uploads.flatMap { upload =>
+        upload.uploadDetails.map { details =>
+          val uploadAlgorithm = if (upload.uploadFields.isDefined) upload.uploadFields.get("x-amz-algorithm") else ""
+          SDESNotification(
+            informationType = "S18",
```
Comment (AndrewRButcher) [inline app/controllers/AppealsController.scala:180]: Can you make this configurable via appConfig?

**Review by AndrewRButcher — CHANGES_REQUESTED** [review]

**Comment by BackRowOla** [comment]
[app-config-base PR](https://github.com/hmrc/app-config-base/pull/6602)

**Review by AndrewRButcher — APPROVED** [review]

**Pull request merged**